doc/getting_started/shift_detection.rst

```diff
@@ -0,0 +1,47 @@
+Distribution Shift Detection
+****************************
+
+Retraining or updating of a machine learning model is usually necessitated by *shifts* in the
+distribution of data that is being served to the model.
+Renate provides methods for distribution shift detection that can help you decide when to update
+your model.
+This functionality resides in the :py:mod:`renate.shift` subpackage.
+
+Shift Detector Interface
+========================
+
+The shift detectors in :py:mod:`renate.shift` derive from a common class
+:py:class:`~renate.shift.detector.ShiftDetector`, which defines the main interface. Once a
+:code:`detector` object has been initialized, one calls :code:`detector.fit(dataset_ref)` on a
+reference dataset (a PyTorch dataset object). This reference dataset characterizes the expected
+data distribution. It may, e.g., be the validation set used during the previous fitting of the
+model. Subsequently, we can score one or multiple query datasets using the
+:code:`detector.score(dataset_query)` method. This method returns a scalar distribution shift score.
+We use the convention that high scores indicate a likely distribution shift. For all currently
+available models, this score lies between 0 and 1.
+
+Available Methods
+=================
+
+At the moment, Renate provides two method for covariate shift detection
```

Comment:
It would be good to provide refs explaining different kind of shifts

Reply:
I'm adding a subsection on shift types.